packages/teams-js/src/public/interfaces.ts

```diff
@@ -749,6 +749,43 @@ export interface DeepLinkParameters {
   subEntityWebUrl?: string;
 }
 
+/**
+ * Shared Dialog Properties
+ */
+export interface BaseDialogInfo {
+  /*
+   * The requested size of the dialog
+   */
+  size: DialogSize;
+
+  /**
+   * Title of the task module.
+   */
+  title?: string;
+}
+
+/**
+ * Data structure to describe dialog information needed to open a adaptive card based dialog.
+ */
+export interface AdaptiveCardDialogInfo extends BaseDialogInfo {
+  /**
+   * JSON defining an adaptive card.
+   */
+  card?: string;
```

Comment:
Why is card optional here?

Reply:
Fixed in latest commit

Reply:
What scenario are you thinking? May be that back compat issue can affect the url dialog since we made the url required in UrlDialogInfo interface

Reply:
Fixed in latest commit